index.html

```diff
@@ -893,9 +893,12 @@
 This is a special case of the more general principle that data should not be used for more
 [=purposes=] than the data's subjects understood it was being collected for.
 
-A service should explain how it uses people's data to protect them and other people, and
-how it might additionally use someone's data if it believes that person has broken the
-rules.
+Services sometimes use people's data in order to protect those or other people.
+This is legitimate as long as the extra risk created by the data collection is
+less than the protection it offers and the extra risk isn't concentrated on
+[=vulnerable people=]. A service that does this should explain what data it's
```

Comment:
I also got encouraged to make this even stronger, citing the [GDPR's](https://www.privacy-regulation.eu/en/recital-47-GDPR.htm) 

> The processing of personal data strictly necessary for the purposes of preventing fraud also constitutes a legitimate interest of the data controller concerned.

and the [CCPA's](https://www.oag.ca.gov/sites/all/files/agweb/pdfs/privacy/oal-sub-final-text-of-regs.pdf)

> A service provider shall not retain, use, or disclose personal information obtained in the
course of providing services except: ... To detect data security incidents or protect against fraudulent or illegal activity;